### DISCUSSION TITLE
Super XBR

### DISCUSSION THREAD

**Discussion by DemitriLatet** [Blinue/Magpie#604] [Ideas]
I have realized that the shader that works best for me in RetroArch for retro-looking upscaling is one called "Super XBR". Specifically, it's a whole set of shaders and configurations included in a preset (MBZ__1__ADV-SUPER-XBR__GDV.slangp) from a pack called MegaBezel. Will it be possible to include Super XBR in the future?

**Comment by Blinue** [comment]
This depends on the difficulty of porting and its popularity. Where can I find its source code?

**Reply by laserboy1** [reply]
https://github.com/n0spaces/py-super-xbr maybe this is
